HtmlMesh/src/html-mesh.ts

```diff
@@ -195,11 +308,9 @@ export class HtmlMesh extends Mesh {
             return;
         }
 
-        const childElement = this._element!.firstElementChild as HTMLElement;
-        if (childElement) {
-            childElement.style.width = `${width}px`;
-            childElement.style.height = `${height}px`;
-        }
+        const sizeFitHandler = SizeFitHandlerMap[this._sizeFit]
```

Comment:
This all gets replaced with `this._fitStrategy.updateSize(this._element, width, height)`